fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -316,14 +323,18 @@ def draw(self, context):
 
 		col.operator(OOT_ImportDL.bl_idname)
 
-		prop_split(col, context.scene, 'ootDLImportName', "DL")
+		if not context.scene.ootDLImportAll:
+			prop_split(col, context.scene, 'ootDLImportName', "DL")
 		if context.scene.ootDLImportUseCustomPath:
 			prop_split(col, context.scene, 'ootDLImportCustomPath', "File")
 		else:		
 			prop_split(col, context.scene, 'ootDLImportFolderName', "Object")
 		prop_split(col, context.scene, "ootDLImportDrawLayer", "Import Draw Layer")
+		if context.scene.ootDLImportAll:
+			col.label(icon = 'INFO', text = 'May have to manually fix layers after.')
 
 		col.prop(context.scene, "ootDLImportUseCustomPath")
+		col.prop(context.scene, "ootDLImportAll")
```

Comment:
it's not obvious that info line has to do with ootDLImportAll imo, I'd suggest
```suggestion
		col.prop(context.scene, "ootDLImportUseCustomPath")
		if context.scene.ootDLImportAll:
			box = col.box()
			box.prop(context.scene, "ootDLImportAll")
			box.label(icon = 'INFO', text = 'May have to manually fix layers after.')
		else:
			col.prop(context.scene, "ootDLImportAll")
```

Reply:
I disagree, the warning is a warning about the layer selection box (that there's no way to set that box which will be fully correct), that's why I put it below that box. I think the fact that that warning only appears when you check the import all box is enough indication to the user that it's a result of that. The text could be better but it'll get cut off depending on how wide the user keeps the pane.

Reply:
You could expand the explanation in the description for the `ootDLImportAll` property, and redirect to it here maybe

Reply:
Sure.